src/core/sys/darwin/sys/event.d

```diff
@@ -43,7 +43,6 @@ enum : short
     EVFILT_FS       =  -9, /* filesystem events */
     EVFILT_USER     = -10, /* User events */
     EVFILT_VM       = -12, /* virtual memory events */
-    EVFILT_SYSCOUNT =  11
```

Comment:
The reasons why I didn't are:
- It's essentially a [version number](https://github.com/opensource-apple/xnu/blob/8cf668b09f7c419fadf6081d1f1bdd6c5033e708/bsd/kern/kern_event.c#L346):
  Any value we define is wrong on some (supported) system, so unless we grab the correct value from the OS at compile time (which would require the D frontend to be able to parse C header files), we should simply not expose it.  It also follows that any D code depending on `EVFILT_SYSCOUNT`'s value being correct for the current OS is broken and should not compile.
- It's used internally only (e.g. [here](https://github.com/opensource-apple/xnu/blob/8cf668b09f7c419fadf6081d1f1bdd6c5033e708/bsd/kern/kern_event.c#L2491)), AFAICT you can't use it in API calls.

So essentially, if you were using `EVFILT_SYSCOUNT` from D, your code was already broken.